test/testlib/nuodb_admin_utilities.go

```diff
@@ -124,7 +124,7 @@ func StartAdminTemplate(t *testing.T, options *helm.Options, replicaCount int, n
 		AddTeardown(TEARDOWN_ADMIN, func() {
 			_, err := k8s.GetPodE(t, kubectlOptions, adminName)
 			if err != nil {
-				t.Logf("Admin pod '%s' is not available and logs can not be retrieved")
+				t.Logf("Admin pod '%s' is not available and logs can not be retrieved", adminName)
```

Comment:
this should return here and not proceed to the other two steps on lines 130-131. I mean, if there is no pod, there is no point in executing those.

Reply:
All I did was add the Pod name that has a placeholder in the format string.

Reply:
I know, but the lack of a return seems to be the reason why you had to make all those changes in GetAppLog

Reply:
The Pod existed but was in PodInitializing state.

Reply:
while you are here, can you please also add a return :)?

Reply:
Will do.